sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2-beta/async_samples/sample_analyze_general_documents_async.py

```diff
@@ -88,6 +88,11 @@ async def analyze_general_documents():
     print("----Entities found in document----")
     for entity in result.entities:
         print("Entity of category '{}' with sub-category '{}'".format(entity.category, entity.sub_category))
+        # NOTE: Calling get_lines() on each entity will return a list of DocumentLines that can be processed 
```

Comment:
I'm thinking we should have some docs about these methods on the README? Then we can point to that section for more information about these convenience methods.

Reply:
Good idea to add usage in examples on Readme, but not sure what you mean about docs on readme.

Also wondering if we should say in the comment above about how the DocumentLines returned are the lines that make up the entity. Not sure it that's immediately clear.

Reply:
Added info on the README, when I said docs on the README I just meant adding info there :P. Done